SampleApp/FocusSquare.swift

```diff
@@ -239,25 +232,22 @@ class FocusSquare: SCNNode {
 		
 		if flash {
 			let waitAction = SCNAction.wait(duration: animationDuration * 0.75)
-			let fadeInAction = SCNAction.fadeOpacity(to: 0.25, duration: animationDuration * 0.125)
-			let fadeAction = SCNAction.fadeOpacity(to: 0.0, duration: animationDuration * 0.125)
-            if let sequence = SCNAction.sequence([waitAction, fadeInAction, fadeAction]) {
-                fillPlane?.runAction(sequence)
-            }
-			
-			let flashSquareAction = flashAnimation(duration: animationDuration * 0.25)
-            if let sequence = SCNAction.sequence([waitAction, flashSquareAction]) {
-                segments?[0].runAction(sequence)
-                segments?[1].runAction(sequence)
-                segments?[2].runAction(sequence)
-                segments?[3].runAction(sequence)
-                segments?[4].runAction(sequence)
-                segments?[5].runAction(sequence)
-                segments?[6].runAction(sequence)
-                segments?[7].runAction(sequence)
-            }
-			
-		}
+            let fadeInAction = SCNAction.fadeOpacity(to: 0.25, duration: animationDuration * 0.125)
```

Comment:
Nit: The indentation is broken around here. Would you mind fixing it (Ctrl + I).

Reply:
Good morning, could you check, because it looks to me the entire file indentation as some issue. 
Did a "Re-indent" on the entire file and this is what I am getting: (PS. I don't have any indention preference setup)
https://github.com/charlymr/ARCharts/commit/1f259ca66076a0b57b9d3e0b3d9d7ac40290fac7